x/evm/module.go

```diff
@@ -32,6 +32,7 @@ const consensusVersion = 7
 var (
 	_ module.AppModule      = AppModule{}
 	_ module.AppModuleBasic = AppModuleBasic{}
+	_ module.HasABCIGenesis = AppModule{}
 
 	_ appmodule.HasEndBlocker   = AppModule{}
 	_ appmodule.HasBeginBlocker = AppModule{}
```

Comment:
why don't we have the `_ appmodule.AppModule = AppModule{}` here as well just as for the ERC-20 - we also don't have it e.g. on the Epochs module